### DISCUSSION TITLE
Create migration guides for Discourse (and others)

### DISCUSSION THREAD

**Discussion by tkrunning** [supabase/supabase#9408] [Feature Requests]
Hi I was asked by [Olyno in the Discord help forum](https://discord.com/channels/839993398554656828/1027589013533626378) to create a topic here suggesting that more migration guides be made for people coming from different platforms.

Personally, I've been running a community on an instance of Discourse, and would like to migrate to using Supabase for auth. Ideally I would like our members to be able to log in via Supabase using their current email/password credentials without needing to do a reset.

Discourse has an explanation for how to export the hashed passwords in a PHC format for importing into other systems (e.g. Auth0 seems to support this): https://meta.discourse.org/t/export-password-hashes-in-the-phc-format/133805/1

Is this something that's possible to import into Supabase as well? If so it would be great having a guide for how to do this!

While I'm personally interested in Discourse (and I've seen others in the Discord asking about it as well), there are probably many other systems it's worth creating migration guides for as well.

**Comment by Olyno** [comment]
Note that a "Migrate" section already exists (e.g [Heroku migration](https://supabase.com/docs/guides/migrations/heroku)). The idea would be to develop this one by adding other guides such as a migration for Discourse, or any other tools.
